connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java

```diff
@@ -192,28 +188,11 @@ private Long ensureLeaderConfig(long maxOffset, WorkerCoordinator coordinator) {
         ConnectorsAndTasks activeAssignments = assignment(memberConfigs);
         log.debug("Active assignments: {}", activeAssignments);
 
-        // This means that a previous revocation did not take effect. In this case, reset
-        // appropriately and be ready to re-apply revocation of tasks
```

Comment:
After this fix, we can work well under this case: _a previous revocation did not take effect_, by computing the totalTasksWeHave / totalWorkers in `performTasksRevocation` method